tests/scripts/analyze_outcomes.py

```diff
@@ -61,24 +61,32 @@ def analyze_coverage(results, outcomes):
             # fixed this branch to have full coverage of test cases.
             results.warning('Test case not executed: {}', key)
 
-def analyze_driver_vs_reference(outcomes, component_ref, component_driver, ignored_tests):
+def analyze_driver_vs_reference(outcomes, component_ref, component_driver,
+                                ignored_suites, ignored_test=None):
     """Check that all tests executed in the reference component are also
     executed in the corresponding driver component.
-    Skip test suites provided in ignored_tests list.
+    Skip:
+    - full test suites provided in ignored_suites list
+    - only some specific test inside a test suite, for which the corresponding
+      output string is provided
     """
     available = check_test_cases.collect_available_test_cases()
     result = True
 
     for key in available:
         # Skip ignored test suites
-        test_suite = key.split(';')[0] # retrieve test suit name
-        test_suite = test_suite.split('.')[0] # retrieve main part of test suit name
-        if test_suite in ignored_tests:
+        full_test_suite = key.split(';')[0] # retrieve full test suit name
+        test_string = key.split(';')[1] # retrieve the text string of this test
+        test_suite = full_test_suite.split('.')[0] # retrieve main part of test suit name
```

Comment:
Typo, pre-existing: test suitE name

Reply:
if the CI is OK and in case this PR gets another approval, can I fix this typo in one of the following PRs of this EPIC?

Reply:
Sure! (As a general rule, when I marke as approved despite comments, it means that: the comments can be addressed if there's another reasons to rework (CI, other reviewer), but otherwise they can be left for later.)